.gitignore

```diff
@@ -0,0 +1,68 @@
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+*.ipch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+*.lib
+
+# Executables
+*.exe
+*.out
+*.app
+
+# vim tags
+tags
+.tags
+.*.swp
```

Comment:
I think that in their time, Vim and Emacs were truly amazing tools, but do we really need to have them here?

Reply:
`.swp` might be useful, that happens from time to time for me, especially I ssh session got accidently disconnected.

Reply:
Well I still use emacs sometimes!  But never vim!!!